com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ConfigurationEntriesCloudModelBuilderTest.java

```diff
@@ -23,28 +24,28 @@
         return Arrays.asList(new Object[][] {
 // @formatter:off
             // (0) Test that provided dependencies from version 1 are public by default:
-            {
-                "mtad-07.yaml", 1, "some-org", "some-space", new Expectation(Expectation.Type.RESOURCE, "expected-configuration-entries-00.json"),
-            },
+//            {
```

Comment:
this is dead code with no chance of surviving - it should be deleted along with the descriptors it's referencing